### PR TITLE
Download: Add Horizon OS APK and Horizon Store app, minor refactoring

### PR DESCRIPTION
**Pull request by akien-mga**
*Edit:* Removed that first change from the PR.
image

image

### CODE REVIEW

**Comment by akien-mga** [comment]
Actually looking at my first screenshot, I made a mistake as it's linking the Horizon Store app on the `/download/android` page which is for 4.3-stable (but we have 4.4-dev2 on Horizon Store).

I'll remove it for now, and we should remember to add it when we release 4.4-stable.

**Comment by akien-mga** [comment]
Added a fix for that stray dot too:
image

File: _data/download_platforms.yml [hunk 1]
```diff
@@ -24,64 +24,74 @@
...
+    - Meta Quest 3 & Pro
+    - arm64
+
 - name: "linux.32"
   title: "Linux"
-  caption: "Standard (x86_32)"
+  caption: "x86_32"
   tags:
     - x86_32
-    - 32 bit
```
Comment (akien-mga) [inline _data/download_platforms.yml:40]: I removed the "32 bit" / "64 bit" tags, which don't say much in an age where most platforms are 64 bit and the relevant distinction is x86_64 or arm64.

File: _data/download_platforms.yml [hunk 2]
```diff
@@ -24,64 +24,74 @@
...
+  title: "Horizon OS"
+  caption: "Horizon Store"
+  tags:
+    - Horizon Store
+    - Meta Quest 3 & Pro
+    - arm64
+
 - name: "linux.32"
   title: "Linux"
-  caption: "Standard (x86_32)"
```
Comment (akien-mga) [inline _data/download_platforms.yml:37]: I removed the "Standard" which didn't say anything useful. It's normally used to distinguish between "Standard" (no C#) and ".NET" (C#) builds, but the way this is designed, .NET builds were actually listed as "Standard - .NET" so all builds were "Standard".

**Review by coppolaemilio — APPROVED** [review]

**Comment by coppolaemilio** [comment]
thanks!

**Pull request merged**